### ISSUE TITLE
India retention - update onboarding (follow up to #24111)

### ISSUE THREAD

**Issue by kjozwiak** [brave/brave-browser#24677]
## Description 

Looks like the body text within the `default browser` screen of onboarding was never updated to https://github.com/brave/brave-browser/issues/24111#issuecomment-1190962959. Looks like it's still using the older text/strings that we've been using.

One way to tell is that the body is translated and all the new strings that we've added into `1.44.x` still haven't been translated. So that's a good indicator that it was never changed.

## Steps to reproduce 

   1. install a Nightly version of Brave that includes https://github.com/brave/brave-core/pull/14327
   2. you'll notice that the text within the body of the `default browser` page doesn't match https://github.com/brave/brave-browser/issues/24111#issuecomment-1190962959

## Actual result 

**`Example`** | **`Example`**
---------------|-----------------
![Screenshot_20220815-145144](https://user-images.githubusercontent.com/2602313/184703252-1b64bddf-4cfd-400d-8cec-c0f5f2052c2b.png) | ![Screenshot_20220815-145631](https://user-images.githubusercontent.com/2602313/184703278-75774d2b-b50e-4677-8723-e0cb0bf49fbf.png)

## Expected result

![image](https://user-images.githubusercontent.com/2602313/184703112-a18c86c2-1025-4dd1-b7b7-e220db7097a2.png)

## Issue reproduces how often 

100% reproducible using the STR/Cases outlined above.

## Version/Channel Information:

- Can you reproduce this issue with the current Play Store version? `N/A`
- Can you reproduce this issue with the current Play Store Beta version? `N/A` (changes haven't been uplifted into `1.43.x` yet)
- Can you reproduce this issue with the current Play Store Nightly version? `Yes`

## Device details
   - Install type (ARM, x86): `ARM`
   - Device type (Phone, Tablet, Phablet): `Pixel 6`
   - Android version: `Android 13`
   
## Brave version

```
Brave | 1.44.43 Chromium: 104.0.5112.81 (Official Build) canary (32-bit)
--- | ---
Revision | 5b7b76419d50f583022568b6764b630f6ddc9208-refs/branch-heads/5112@{#1309}
OS | Android 13; Build/TPB4.220624.008
```

### Website problems only
- Does the issue resolve itself when disabling Brave Shields? `N/A`
- Does the issue resolve itself when disabling Brave Rewards? `N/A`
- Is the issue reproducible on the latest version of Chrome? `N/A`

**Comment by kjozwiak** [comment]
The above will require `1.43.72` or higher for `1.43.x` verification. We'll also need to check if these new strings were translated once we get translations merged into `1.43.x.`. Believe we should be ordering the new translations sometime early next week.

**Comment by stephendonner** [comment]
## Verified `PASSED` using 

Brave `1.43.74, Chromium 104.0.5112.102` on a `Google Pixel XL` running `Android 9.0`.

NOTE: copy will be verified over in https://github.com/brave/brave-browser/issues/24111, once we get translated strings, as @kjozwiak mentions above.

Set my system language in Android to Hindi:

1.43.74 | Hindi
--------|------
![Screenshot_20220818-133425](https://user-images.githubusercontent.com/387249/185491287-11511120-4491-48f3-8013-a273a4bddc3f.png) | ![Screenshot_20220818-134147](https://user-images.githubusercontent.com/387249/185491289-d4b12ef5-ac08-429c-bf91-2e182c19616b.png)

### Hindi

#### Confirmed new India-onboarding-specific strings were used (not all translated; will be verified over in https://github.com/brave/brave-browser/issues/24111

onboarding 1 | onboarding 2 | onboarding 3
-------------|---------------|--------------
![Screenshot_20220818-132924](https://user-images.githubusercontent.com/387249/185491722-0594f7e4-e0d8-4bf9-a904-35f555066121.png) | ![Screenshot_20220818-132931](https://user-images.githubusercontent.com/387249/185491733-f0467d94-bc5d-4ff6-b020-e38bea6e9d78.png) | ![Screenshot_20220818-133214](https://user-images.githubusercontent.com/387249/185491737-22ad16ee-0615-439e-86c6-4210bfe521d8.png)



### English (United States)

#### Confirmed existing strings were used/retained

onboarding 1 | onboarding 2 | onboarding 3
--------------|--------------|--------------
![Screenshot_20220818-133911](https://user-images.githubusercontent.com/387249/185492114-f896338e-6805-400a-9777-f9711e30f474.png) | ![Screenshot_20220818-133921](https://user-images.githubusercontent.com/387249/185492126-225244e2-5a21-41b5-aa47-23819ed56fb8.png) | ![Screenshot_20220818-133931](https://user-images.githubusercontent.com/387249/185492132-cd204b4e-9053-44d2-a1a7-74c8b50a321a.png)
